app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+
+    if user&.authenticate(params[:session][:password])
+      # Log the user in and redirect to the user's show page.
+      cookies.permanent[:user_id] = user.id
+      cookies.permanent[:remember_token] = user.remember_token
```

Comment:
as said by @Ispirett 
> You should be giving the User a new remember token here [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_method_requirement.png)
> eg:
> 
> ```
>  remember_token = User.generate_remember_token
>   user.update_attribute(:remember_token, remember_token)
>   cookies.permanent[:remember_token] = remember_token
> ```


You can check the requirement in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_method_requirement.png)